build/ci/evergreen.yml

```diff
@@ -167,6 +167,29 @@ functions:
           set -Eeou pipefail
 
           curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh
+  "install snyk":
+    - command: shell.exec
+      type: setup
+      params:
+        working_dir: src/github.com/mongodb/mongocli/bin
+        script: |
+          set -Eeou pipefail
+
+          curl -sfL https://github.com/snyk/snyk/releases/download/v${snyk_ver}/snyk-linux -o snyk
+          chmod +x snyk
+  "snyk scan":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src/github.com/mongodb/mongocli
+        script: |
+          export SNYK_TOKEN=${snyk_token}
+
+          set -Eeou pipefail
+
+          export GOROOT="${go_root}"
+          export PATH="./bin:$GOROOT/bin:$PATH"
```

Comment:
is GOROOT needed?

Reply:
I left them because I'm not sure if snyk needs a working go env to run, if there's no `GOROOT` set go complains a lot, it has a default usually but that's not set on evergreen

Reply:
Yes, goroot is needed, test [here](https://spruce.mongodb.com/task/mongocli_master_code_health_snyk_scan_patch_9ec0ac69775c32c2fb9b320f23caeddf1b13b4df_5f22877230661517184899d1_20_07_30_08_43_58/logs) without it